patches/react-select+5.8.0.patch

```diff
@@ -0,0 +1,137 @@
+diff --git a/node_modules/react-select/.DS_Store b/node_modules/react-select/.DS_Store
```

Comment:
These pointers needs to be confirmed with Vivek:
- Since the patch is added in all the repos, it is a overhead in case of version updates
- If react-select changes this file, our patch would be corrupted
- How about hard coding the react select version instead of using the ^

Reply:
@vivek-devtron please confirm this

Reply:
We should fix the reactSelect version. I guess we discussed the same that day. when you shared the approach @RohitRaj011

Reply:
react-select version is already fixed